index.ts

```diff
@@ -351,11 +361,16 @@ class Snowflake {
         )
     }
 
+    async appendToFilesList(storage: StorageExtension, fileName: string) {
```

Comment:
this is the type of thing we should probably make part of the API. Some functions for using lists that a lot of plugins could probably use.

unrelated here though

Reply:
agreed, and agreed on not now.